pages/api/content/glamorous.js

```diff
@@ -1,6 +1,131 @@
+
+    ~~~js
+    const MyComponent = props => <div {...props} />
+    const myGlamorousComponentFactory = glamorous(
+      MyComponent,
+      {displayName: 'MyGlamorousComponent'}
+    )
+    ~~~
+    
+    >Note: the ~displayName~ can also included in the className that your components are given which makes 
+    >the development experience a bit nicer. To enable this see the section about ~config~. This will likely 
+    >be enabled by default in the next major change.
+    
+    And now all components created by the ~myGlamorousComponentFactory~ will have the ~displayName~ of 
+    ~MyGlamorousComponent~.
+
+    There is also a [babel plugin](https://www.npmjs.com/package/babel-plugin-glamorous-displayname) that can 
+    monkey-patch the ~displayName~ onto the components that you create from your component factory.
+    
+    ### rootEl
+    
+    React has an [Unknown Prop Warning](https://facebook.github.io/react/warnings/unknown-prop.html) that it 
+    logs when you pass spurious props to DOM elements: (i.e. ~<div big={true} />~). Because you can style your 
+    components using props, glamorous needs to filter out the props you pass so it doesn't forward these on to 
+    the underlying DOM element. However, if you create your own factory using a custom component, glamorous will 
+    just forward all the props (because the component may actually need them and glamorous has no way of knowing). 
+    But in some cases, the component may be spreading all of the props onto the root element that it renders. 
+    For these cases, you can tell glamorous which element is being rendered and glamorous will apply the same logic 
+    for which props to forward that it does for the built-in factories. For [example](https://codesandbox.io/s/P18oV4kD2):
+
+    ~~~js
+    const MyComponent = props => <div {...props} />
+    const myGlamorousComponentFactory = glamorous(
+      MyComponent,
+      {rootEl: 'div'}
+    )
+
+    const MyGlamorousComponent = myGlamorousComponentFactory(props => ({
+      fontSize: props.big ? 36 : 24,
+    }))
+
+    <MyGlamorousComponent big={true} id="room423" />
+    // this will render:
+    // <div id="room423" />
+    // with {fontSize: 36}
+    // ~big~ is not forwarded to MyComponent because the ~rootEl~ is a ~div~ and ~big~
+    // is not a valid attribute for a ~div~ however ~id~ will be forwarded because
+    // ~id~ is a valid prop
+    ~~~
+    ### forwardProps
+
+    There are some cases where you're making a ~glamorousComponentFactory~ out of a custom component that spreads 
+    some of the properties across an underlying DOM element, but not all of them. In this case you should use 
+    ~rootEl~ to forward only the right props to be spread on the DOM element, but you can also use ~forwardProps~ 
+    to specify extra props that should be forwarded. For [example](https://codesandbox.io/s/GZEo8jOyy:
```

Comment:
A typo in the link, it should be `[example](https://codesandbox.io/s/GZEo8jOyy)`